### PR TITLE
Feature/project setup

### PR DESCRIPTION
**Pull request by francisuloko**
PR Summary:

1. Created App using Create React App (CRA )
2. Setup Github workflows and Linters
3. Added project description README
4. Ran App without errors

### CODE REVIEW

File: src/reportWebVitals.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+      getCLS(onPerfEntry);
+      getFID(onPerfEntry);
+      getFCP(onPerfEntry);
+      getLCP(onPerfEntry);
+      getTTFB(onPerfEntry);
+    });
+  }
+};
+
+export default reportWebVitals;
```
Comment (DeliceLydia) [inline src/reportWebVitals.js:15]: - [ ] Please delete all the boilerplates that came with installing with the application that you will not use 
- [ ] Even in the `public` folder you can just keep `index.html`

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
```
Comment (DeliceLydia) [inline src/index.js:5]: - [ ] This `reportWebVitals` is not needed here

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>,
+  document.getElementById('root'),
+);
+
+// If you want to start measuring performance in your app, pass a function
+// to log results (for example: reportWebVitals(console.log))
+// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
+reportWebVitals();
```
Comment (DeliceLydia) [inline src/index.js:17]: - [ ] even this

File: src/setupTests.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
+// expect(element).toHaveTextContent(/react/i)
+// learn more: https://github.com/testing-library/jest-dom
+import '@testing-library/jest-dom';
```
Comment (DeliceLydia) [inline src/setupTests.js:5]: - [ ] All of this are not needed for now

File: src/App.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+          rel="noopener noreferrer"
+        >
+          Learn React
+        </a>
+      </header>
+    </div>
+  );
+}
+
+export default App;
```
Comment (DeliceLydia) [inline src/App.js:29]: - [ ] You can keep `app.js` but try to remove everything that is not needed in the app

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @francisuloko, 

Good job so far! there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 6]
```diff
@@ -1 +1,99 @@
...
+
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+
+### `yarn build`
+
+Builds the app for production to the `build` folder.\
+It correctly bundles React in production mode and optimizes the build for the best performance.
+
+The build is minified and the filenames include the hashes.\
```
Comment (Dipeshtwis) [inline README.md:27]: [OPTIONAL]

- I think you have missed the project description in the README. Please add to make README professional.

**Review by Dipeshtwis — APPROVED** [review]
# Hi @francisuloko, 

### Status: Approved ✔️

:+1:  You have implemented all the previous reviewer suggestions. Well Done.

Your project is complete!  There are some optional changes I've suggested to you to make your project best, other than that nothing else to say.
Congratulations! 🎉 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

**_Check the comments under the review_**


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Dipesh Kumar_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 6 above)
Comment (francisuloko) [reply README.md:27]: Okay, this is noted.

**Comment by francisuloko** [comment]
Thank you for reviewing and approving my code. Happy Coding @Dipeshtwis @DeliceLydia

**Pull request merged**